diff -Nuar a/SConstruct b/SConstruct
--- a/SConstruct	2020-04-05 19:19:54.000000000 +0000
+++ b/SConstruct	2021-03-12 05:43:00.158534286 +0000
@@ -832,7 +832,7 @@
         ])
 
         for lib in ['libpulsecore-*.so', 'libpulsecommon-*.so']:
-            path = '%s/src/.libs/%s' % (pa_build_dir, lib)
+            path = '%s/%s' % (pa_build_dir, lib)
             libs = env.Glob(path)
             if not libs:
                 env.Die("can't find %s" % path)
